pagefaultgames/pokerogue
# [Balance] Remove Species Locks from Aura Wheel and Poison Puppeteer

## What are the changes the user will see?
- Aura Wheel is now functional for species other than Morpeko (and fusions with Morpeko). It defaults to Electric type when used by Pokemon other than Morpeko
- Poison Puppeteer is now functional for species other than Pecharunt

## Why am I making these changes?
These species locks are unnecessarily restrictive with the existence of fusions, egg moves, and passive abilities.

## What are the changes from a developer perspective?
- `data/move`: Removed Aura Wheel's condition requiring the user to be Morpeko (or a Morpeko fusion)
- `data/ability`: Made Poison Puppeteer's effect attribute unconditional to the source's species.
- Localization changes:
  - Replaced reference to Pecharunt in Poison Puppeteer's description with "this Pokémon"
  - Updated Aura Wheel's description to reflect the changes to species-related conditions

### Screenshots/Videos



## How to test the changes?
Using `src/overrides.ts`:
- Give a Pokemon other than Pecharunt Poison Puppeteer and Toxic. You should see that Toxic also confuses the target.
- Give a Pokemon other than Morpeko Aura Wheel. You should see that the move doesn't fail and defaults to Electric type.

## Checklist
